@JunTaoLuo will need another AspNetCore-pr-validation-temp retry once Linux build completes:
```
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(4551,5): warning MSB3026: Could not copy "F:\vsagent\82\s\src\Servers\IIS\AspNetCoreModuleV2\InProcessRequestHandler\bin\Release\Win32\aspnetcorev2_inprocess.dll" to "F:\vsagent\82\s\src\Servers\testassets\ServerComparison.TestSites\bin\Release\net461\x86\aspnetcorev2_inprocess.dll". Beginning retry 1 in 1000ms. The process cannot access the file 'F:\vsagent\82\s\src\Servers\testassets\ServerComparison.TestSites\bin\Release\net461\x86\aspnetcorev2_inprocess.dll' because it is being used by another process.  [F:\vsagent\82\s\src\Servers\testassets\ServerComparison.TestSites\ServerComparison.TestSites.csproj]
C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\MSBuild\15.0\Bin\Microsoft.Common.CurrentVersion.targets(4551,5): warning MSB3026: Could not copy "F:\vsagent\82\s\src\Servers\IIS\AspNetCoreModuleV2\InProcessRequestHandler\bin\Release\Win32\aspnetcorev2_inprocess.dll" to "bin\Release\netcoreapp2.2\x86\aspnetcorev2_inprocess.dll". Beginning retry 1 in 1000ms. The process cannot access the file 'bin\Release\netcoreapp2.2\x86\aspnetcorev2_inprocess.dll' because it is being used by another process.  [F:\vsagent\82\s\src\Servers\IIS\IIS\test\IIS.Tests\IIS.Tests.csproj]
F:\vsagent\82\s\build\RepositoryBuild.targets(143,5): error : Building EntityFrameworkCore failed: build.cmd exited code 1 [F:\vsagent\82\.dotnet\buildtools\korebuild\2.2.1-build-20190219.3\KoreBuild.proj]
```

--- a/.azure/pipelines/jobs/default-build.yml
+++ b/.azure/pipelines/jobs/default-build.yml
@@ -137,7 +137,7 @@ jobs:
         feedSource: https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
   - ${{ parameters.beforeBuild }}
   - ${{ if eq(parameters.agentOs, 'Windows') }}:
-    - script: .\$(BuildDirectory)\build.cmd -ci /p:SignType=$(_SignType) /p:Configuration=$(BuildConfiguration) $(BuildScriptArgs)
+    - script: .\$(BuildDirectory)\build.cmd -ci /p:SignType=$(_SignType) /p:Configuration=$(BuildConfiguration) $(BuildScriptArgs) /v:diag
       displayName: Run build.cmd
     - powershell: eng\scripts\KillProcesses.ps1
       displayName: Kill processes

--- a/.azure/pipelines/jobs/default-build.yml
+++ b/.azure/pipelines/jobs/default-build.yml
@@ -137,7 +137,7 @@ jobs:
         feedSource: https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
   - ${{ parameters.beforeBuild }}
   - ${{ if eq(parameters.agentOs, 'Windows') }}:
-    - script: .\$(BuildDirectory)\build.cmd -ci /p:SignType=$(_SignType) /p:Configuration=$(BuildConfiguration) $(BuildScriptArgs) /v:diag
+    - script: .\$(BuildDirectory)\build.cmd -ci /p:SignType=$(_SignType) /p:Configuration=$(BuildConfiguration) $(BuildScriptArgs)
       displayName: Run build.cmd
     - powershell: eng\scripts\KillProcesses.ps1
       displayName: Kill processes

--- a/build/RepositoryBuild.targets
+++ b/build/RepositoryBuild.targets
@@ -129,7 +129,7 @@
     <Message Text="============ Building $(RepositoryToBuild) ============" Importance="High" />
 
     <Exec
-      Command="./$(_BuildScriptToExecute) -Path $(BuildRepositoryRoot) $(BuildArguments)"
+      Command="./$(_BuildScriptToExecute) -Path $(BuildRepositoryRoot.TrimEnd('\')) $(BuildArguments)"
       IgnoreStandardErrorWarningFormat="true"
       WorkingDirectory="$(RepositoryRoot)"
       IgnoreExitCode="true"

--- a/build/dependencies.props
+++ b/build/dependencies.props
@@ -28,7 +28,7 @@
     <MicrosoftAspNetCoreBuildToolsApiCheckPackageVersion>$(KoreBuildVersion)</MicrosoftAspNetCoreBuildToolsApiCheckPackageVersion>
     <InternalAspNetCoreSdkPackageVersion>$(KoreBuildVersion)</InternalAspNetCoreSdkPackageVersion>
     <InternalAspNetCoreSiteExtensionSdkPackageVersion>$(KoreBuildVersion)</InternalAspNetCoreSiteExtensionSdkPackageVersion>
-    <InternalAspNetCoreSdkPackageVersion Condition=" '$(KoreBuildVersion)' == '' ">2.2.1-build-20190110.1</InternalAspNetCoreSdkPackageVersion>
+    <InternalAspNetCoreSdkPackageVersion Condition=" '$(KoreBuildVersion)' == '' ">2.2.1-build-20190219.3</InternalAspNetCoreSdkPackageVersion>
   </PropertyGroup>
 
   <!-- These are package versions that should not be overridden or updated by automation. -->

